src/components/notes-form/NotesForm.jsx

```diff
@@ -94,6 +97,14 @@ export const NotesForm = ({ closeForm, isFromEdit, editNoteData }) => {
         </div>
         <div className="form--cta">
           <div className="toolbar">
+            <PriorityDropdown
+              color={notesData.color}
```

Comment:
you could destructure notesData and then use it, since its getting repeated multiple times